lib/openhab/dsl/items/builder.rb

```diff
@@ -448,7 +450,12 @@ def channel(channel, config = {})
         #   expire 5.minutes, state: NULL
         # @example Send a command on expiration
         #   expire 5.minutes, command: OFF
-        def expire(*args, command: nil, state: nil)
+        # @example Specify the duration and command in the same string
+        #   expire "5h,command=OFF"
+        # @example Set the expire configuration
+        #   expire 5.minutes, ignore_state_updates: true
+        #
+        def expire(*args, command: nil, state: nil, **config)
```

Comment:
I'd rather have explicit kwargs ignore_state_updates and ignore_commands here, that are documented for what they mean

Reply:
The tricky part is, we want to allow the original configuration key names `ignoreStateUpdates` and `ignoreCommands` too. Should we add all 4 kwargs?

Reply:
is there a specific reason we want to allow the original configuration names? if someones wants to do that, they can just set the metadata directly

Reply:
I'll forego the need to accept the camel case version for now. If the needs arise in the future, it can be added.